code/dataobjects/DynamicList.php

```diff
@@ -36,6 +36,43 @@ public function getCMSFields() {
 			$fields->addFieldToTab('Root.Items', new GridField('Items', 'Dynamic List Items', $this->Items(), $conf));
 		}
 
+		// Make sure the draft records are being looked at.
+
+		$stage = Versioned::current_stage();
+		Versioned::reading_stage('Stage');
+		$used = EditableFormField::get()->filter(array(
```

Comment:
This is an explicit binding to the UDF module; is it possible to pull this logic out to an extension that's configured using the yml directives so it's only applied if the UDF module exists?